src/restaurant.test.js

```diff
@@ -0,0 +1,47 @@
+const Restaurant = require('./restaurant')
+
+const menu = {
+  items: {
+    fries: {
+      name: 'fries', price: 1.99
+    },
+    burger: {
+      name: 'burger', price: 3.99
+    }
+  }
+}
+
+const orderDouble = {dishes: []}
+
+beforeEach(() => {
+  restaurant.clearOrder();
+});
+
+const restaurant = new Restaurant("Dishoom", menu, orderDouble)
+
+describe('restaurant', ()=> {
+  describe('listMenu', ()=> {
+    it('returns a hash object', ()=> {
+      expect(typeof(restaurant.listMenu()) == 'object').toBe(true)
```

Comment:
this is an example of testing 'state' that I think is generally to be avoided. Worth asking coaches I think, but when I did something similar Leo mentioned that testing state rather than behaviour.